format/spec.md

```diff
@@ -874,6 +878,11 @@ Maps with non-string keys must use an array representation with the `map` logica
 |**`list`**|`array`||
 |**`map`**|`array` of key-value records, or `map` when keys are strings (optional).|Array storage must use logical type name `map` and must store elements that are 2-field records. The first field is a non-null key and the second field is the value.|
 
+Notes:
+
+1. Avro type annotation `adjust-to-utc` is an Iceberg convention; default value is `false` if not present.
+2. Avro logical type `timestamp-nanos` is an Iceberg convention; the Avro specification does not define this type.
```

Comment:
Is this necessary? Shouldn't we just go add this type to Avro, or use an Avro version?

Reply:
In the Google doc I already suggested adding this to the Avro spec: https://docs.google.com/document/d/1bE1DcEGNzZAMiVJSZ0X1wElKLNkT9kRkk0hDlfkXzvU/edit?disco=AAAA5YbF34Y

Let me sent out an email on the Avro devlist

Reply:
@Fokko please drop a link to Avro devlist post when you have a moment. The mailing list link on [the Avro community page](https://avro.apache.org/community/) is just an email address, can't find the web UI.

Reply:
Ah, we should update that page, that's not helping anyone. You can easily see the devlist on ponymail: https://lists.apache.org/list.html?dev@avro.apache.org

On the bottom-left you can subscribe to the devlist. You can find my message here: https://lists.apache.org/thread/s985tc2m6bj5lvck017q06zkt3dg1f31 or the PR https://github.com/apache/avro/pull/2554. Feel free to chime in anywhere.

Reply:
@rdblue @Fokko I agree that we should ask this type be added to Avro.

It will be months before this change is added to Avro, then supported by dependent projects. Do you propose that:
- We wait for the Avro spec to change before modifying the Iceberg spec?
- Adjust the Iceberg spec language to reflect the intention to switch to the Avro type when it becomes available?

Reply:
Since most Iceberg implementations have their own Avro readers, I think we're fine with running this in parallel. I don't see any huge issues on the Avro side (maybe some discussion around the physical type), but I would not consider that blocking.

Reply:
@Fokko I understand; do I need to change this PR, then?

Reply:
It depends on what comes out of the Avro PR, but I think we're good for now.